csrc/scheduler/normalization.cpp

```diff
@@ -123,20 +123,26 @@ std::shared_ptr<ReductionParams> innerOuterPersistentHeuristic(
   // inner_dim_numel is dividable by the multiplication of a quarter warp and
   // vectorize_factor.
   iop.inner_vect = (int64_t)vectorize_factor;
-  auto opt_inner_batch = normalization_scheduler_utils::
+  const auto& batch_and_block_size = normalization_scheduler_utils::
       getOptionalInnerOuterPersistentBufferBatches(
           inner_dim_numel,
           outer_dim_numel,
           max_persistent_buffer_size,
           iop.inner_vect,
-          dev_prop->warpSize);
+          dev_prop->warpSize,
+          true);
```

Comment:
Here `true` is passed to `getOptionalInnerOuterPersistentBufferBatches`, indicates we must use whatever we get as we are already entering the step to schedule the fusion. We need this because we enforced projection for fp32.

Reply:
Can you comment on why it's necessary despite it's also checked by `canScheduleRuntime`?

Reply:
added